docs/en/latest/FAQ.md

```diff
@@ -88,3 +88,12 @@ You can change the Admin API key in two ways:
 
 1. Modify the key in both [apisix/values.yaml](https://github.com/apache/apisix-helm-chart/blob/57cdbe461765cd49af2195cc6a1976cc55262e9b/charts/apisix/values.yaml#L181) and [apisix/apisix-ingress-controller/values.yaml](https://github.com/apache/apisix-helm-chart/blob/57cdbe461765cd49af2195cc6a1976cc55262e9b/charts/apisix-ingress-controller/values.yaml#L128) files.
 2. You can also set this imperatively by passing the flag `--set ingress-controller.config.apisix.adminKey=<new key> --set admin.credentials.admin=<new key>` to the `helm install` command.
+
+
+## Why does my Ingress resource not have an address?
+
+You will need to get the apisix-gateway service an external IP assigned for it to reflect on the Ingress's status.
+1. While installing APISIX helm chart make sure to override gateway type with `--set gateway.type=LoadBalancer`. 
```

Comment:
Setting it as NodePort will also work.

Reply:
But in case of NodePort, an external IP will not be allocated to it, right?

Reply:
yes.

https://github.com/apache/apisix-ingress-controller/blob/e809cfbaaad6f6c3cbd23f5ec8e93bc1d87f543d/conf/config-default.yaml#L44

we can use `ingress_status_address`

see the comment:

>   when there is no available information on the Service
> used for publishing on the data plane,
> the static address provided here will be
>  used to update the status information of Ingress.
>  When ingress-publish-service is specified at the same time, ingress-status-address is preferred.
> For example, no available LB exists in the bare metal environment.

Reply:
Applied suggestion